cmd/kube-apiserver/app/server.go

```diff
@@ -213,6 +213,22 @@ func newEtcd(etcdConfigFile string, etcdServerList util.StringList, storageVersi
 	return master.NewEtcdHelper(client, storageVersion, pathPrefix)
 }
 
+// Tells the service manager like systemd about startup. For further docs:
+// http://www.freedesktop.org/software/systemd/man/sd_notify.html
+func notifyReady() {
```

Comment:
Consider adding an interface to the app framework (the thing that calls the Run() method) that does this--otherwise you'll be copy-pasting this many places.